client/client_test.go

```diff
@@ -2916,13 +2919,9 @@ loop0:
 	require.Equal(t, 0, len(du))
 
 	// examine contents of exported tars (requires containerd)
-	var cdAddress string
-	if cd, ok := sb.(interface {
-		ContainerdAddress() string
-	}); !ok {
+	cdAddress := sb.ContainerdAddress()
+	if cdAddress == "" {
 		return
```

Comment:
could you mind to add some log for skip check? It will be nice to debug!

Reply:
Yeah good call, updated with that